brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Example.scala

```diff
@@ -32,8 +32,17 @@ abstract class CSVJob(args: Args) extends TrainerJob(args) {
     fn
   }
 
-  def train(trainingData: TypedPipe[Instance[String, Double, Map[String, Long]]]): Trainer[String, Double, Map[String, Long]] = {
-    Trainer(trainingData, KFoldSampler(4))
+  private def id(m: DefaultMetadata): String = {
+    m.id
+  }
+
+  def train(trainingData: TypedPipe[Instance[DefaultMetadata, Map[String, Double], Map[String, Long]]]): Trainer[DefaultMetadata, String, Double, Map[String, Long], Unit] = {
+    val annotator = defaultMetadataAnnotator[Unit]
```

Comment:
`Annotator` isn't an implicit so it needs to be passed explicitly... so it's not really a "default"